.github/workflows/ci.yml

```diff
@@ -0,0 +1,21 @@
+name: CI
+on:
+  - push
+  - pull_request
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4.1.0
+      - uses: actions/setup-java@cd89f46ac9d01407894225f350157564c9c7cee2 # v3.12.0
+        with:
+          distribution: temurin
+          java-version: 11
+      - run: sudo apt-get update && sudo apt-get install metacity
```

Comment:
In other projects we used xvfb with Eclipse/SWT builds. Is there a reason to run a full blown x server?

Reply:
@marchof Correct me if I'm wrong, but seems that you're mixing two things here - X server and X window manager. Xvfb is used here - see invocation a few lines below. But AFAIK we also need a window manager - quoting https://wiki.eclipse.org/SWTBot/Automate_test_execution#Linux_hints_2
> start a window manager: metacity is recommended

Reply:
I wasn't aware about the requirement of the X window manager. I never isntalled that for SWT tests. Did you try without?

Reply:
@marchof just tried it... and it works 😅 

Actually I just copied this from our old configuration for Travis CI - see
https://github.com/eclipse/eclemma/blob/7ff40a551893f2fede67839c89989e758621ec9a/.travis.sh#L8

Also there are many statements like
* https://www.eclipse.org/forums/index.php/t/171584/
  > Just a dummy X server with no window manager will not work
* https://www.lorenzobettini.it/2015/07/running-swtbot-tests-in-travis/
  > to run SWTBot tests you need a Window Manager
* https://tycho-user.eclipse.narkive.com/3gkGnDj6/run-swtbot-tests-using-tycho-maven#
  > start a window manager in a shell step before starting the tests
  > In EGit we use metacity as the window manager

maybe this was the case for older Eclipse / SWTBot tests... 🤔 
But if nowadays it works, then great, builds will be faster. Thanks! 👍